### PR TITLE
Tickets/sitcom 1344

### PR DESCRIPTION
**Pull request by isotuela**
Hi Karla, 
This ticket contains several commits, related to the restructuring and improvement of the page:
- Change of folder names to facilitate navigation and easier URL; instead of AuxTel/AuxTel-troubleshooting/ATCS-Troubleshooting/ it should be now AuxTel/Troubleshooting/ATCS. Let me know if it makes sense, and I look forward to hearing your thoughts. 
- Also index pages are now called index.rst instead of the longer version that made the URL really heavy and long. 
- Improvements in the main index page, now containing grids of cards with images for the different sections. 
-  In the second level of indexes, for instance, AuxTel now the right hand side contains Titles to facilitate organizing the information. The tree menu in the Observatory-Control-System needs still some work, depending on the content we end up putting in there. 
- Replace MainTel for Simonyi everywhere, and uniformize "Precondition" and "Post-Condition" across all files. Also remove reference to AT, in favour of AuxTel. 
- Move, as you suggested, the Safety-Control-Systems to the Safety Section. The index for Safety also needs more work. 

I'd like to merge this as a base for all the work we are doing, and I believe it's an improvement in navigation. 

Cheers! 
Ioana

### CODE REVIEW

File: AuxTel/Non-Standard-Operations/ATCalSys/index.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
-.. Main-Non-standard-Operations-LaserTracker-Non-Standard-Procedures:
+.. _AuxTel-Non-standard-Operations-ATCalSys:
 
 #############################################
-Laser Tracker Non-standard procedures
+AT Calibration System
```
Comment (kpenaramirez) [inline AuxTel/Non-Standard-Operations/ATCalSys/index.rst:4]: AuxTel?

File: AuxTel/Troubleshooting/ATCalSys/index.rst [hunk 2]
```diff
@@ -1,7 +1,7 @@
-.. MainTel-Troubleshooting-M2-Troubleshooting:
+.. _AuxTel-Troubleshooting-ATCalSys:
 
 #####################################
-M2 Troubleshooting
+AT Calibration System
```
Comment (kpenaramirez) [inline AuxTel/Troubleshooting/ATCalSys/index.rst:4]: AuxTel?

File: Daytime-Nighttime-Interactions/announce-component-usage.rst [hunk 3]
```diff
@@ -37,9 +37,9 @@ Overview
 
 This topic covers how to properly announce the usage of component(s) on the summit.
 
-.. _Announcing-Component-Usage-Prerequisites:
+.. _Announcing-Component-Usage-Precondition:
 
-Prerequisites
+Precondition
 ^^^^^^^^^^^^^
```
Comment (kpenaramirez) [inline Daytime-Nighttime-Interactions/announce-component-usage.rst:43]: ^^^^^ ---> ===== ?

File: Observing-Interface-Setup/dds-interface.rst [hunk 4]
```diff
@@ -11,7 +11,7 @@ There are 3 use-cases for DDS Daemons defined by their location and application.
 #. Local daemons for local CSCs or services which are mostly used for non-production environments (i.e. development)
 #. Nublado daemons which are mainly used as a way to manually ensure system stability for night time operations.
 
-Prerequisites
+Precondition
 ^^^^^^^^^^^^^
```
Comment (kpenaramirez) [inline Observing-Interface-Setup/dds-interface.rst:15]: ^^^^^ --> =====?

File: Observing-Interface-Setup/nublado-notebook-environment-update.rst [hunk 5]
```diff
@@ -50,14 +50,14 @@ By default, the platform provides a set of environments with released software v
 - interfaces (*e.g.*, `xml <https://ts-xml.lsst.io/>`__, `IDL <https://github.com/lsst-ts/ts_idl>`__, `SAL <https://ts-sal.lsst.io/>`__, `Salobj <https://ts-salobj.lsst.io/>`__), and
 - high-level control algorithms (*e.g.*, `ATCS <https://ts-observatory-control.lsst.io/>`__, LATISS, MTCS, ComCam).
 
-.. _Update-Notebook-Environment-in-Nublado-Prerequisites:
+.. _Update-Notebook-Environment-in-Nublado-Precondition:
 
-Prerequisites
+Precondition
 ^^^^^^^^^^^^^
```
Comment (kpenaramirez) [inline Observing-Interface-Setup/nublado-notebook-environment-update.rst:56]: ^^^^^ -> ====?

File: Simonyi/Non-Standard-Operations/ComCam/comcam-cooldown.rst [hunk 6]
```diff
@@ -34,14 +34,14 @@ Overview
 
 This procedure describes the starting up and cooling down of the ComCam cryostat.
 
-.. _ComCam-Cooldown-Prerequisites:
+.. _ComCam-Cooldown-Precondition:
 
-Prerequisites
+Precondition
 ^^^^^^^^^^^^^
```
Comment (kpenaramirez) [inline Simonyi/Non-Standard-Operations/ComCam/comcam-cooldown.rst:40]: ^^^^ --> ====?

File: Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst [hunk 7]
```diff
@@ -38,7 +38,7 @@ AuxTel must be under LOTO (lock-out tag-out) any time physical work is undertake
 When working with environmental sensors, on the truss, mirrors, or anywhere in the dome where unexpected movement could cause a safety hazard, the telescope must be locked out. 
 Trained summit support staff may use this procedure if they meet all of the preconiditon requirements. 
 
-.. _MainTel-LOTO-procedure-Precondition:
+.. _Simonyi-LOTO-procedure-Precondition:
 
 Precondition
 ^^^^^^^^^^^^
```
Comment (kpenaramirez) [inline Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst:44]: ^^^^-->====?

File: Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst [hunk 8]
```diff
@@ -48,7 +48,7 @@ Precondition
 .. If there is a different procedure that is critical before execution, carefully consider if it should be linked within this section or as part of the Procedure section below (or both).
 
 
-.. MainTel-LOTO-procedure-Post-Condition:
+.. _Simonyi-LOTO-procedure-Post-Condition:
 
 Post-Condition
 ^^^^^^^^^^^^^^
```
Comment (kpenaramirez) [inline Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst:54]: Postcondition?
==========?

File: Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst [hunk 9]
```diff
@@ -59,9 +59,9 @@ Post-Condition
...
 
 The drives are powered off and the AuxTel Control cabinet is locked from the outside. 
-The Maintel is under LOTO and is not able to move by any controls.
+The Simonyi Telescope is under LOTO and is not able to move by any controls.
 
-.. _AuxTel-LOTO-procedure-Procedure-Steps:
+.. _Simonyi-LOTO-procedure-Procedure-Steps:
 
 Procedure Steps
 ^^^^^^^^^^^^^^^
```
Comment (kpenaramirez) [inline Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst:67]: ^^^ --> ==== ?

File: Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst [hunk 10]
```diff
@@ -72,7 +72,7 @@ Procedure Steps
 
 
 
-.. _MainTel-LOTO-procedure-Troubleshooting:
+.. _Simonyi-LOTO-procedure-Troubleshooting:
 
 Troubleshooting
 ^^^^^^^^^^^^^^^
```
Comment (kpenaramirez) [inline Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst:78]: ^^^^ ---> ====?

**Comment by kpenaramirez** [comment]
Thank a lot for all the changes. (I scanned only the differences in the files, not every single line as usual).
- Most of the comments are related to file heading structure, not sure which type of heading do you want to preserve. 
- All the "Precondition" seems now to be homogeneous along the text, I only saw one change in one template. Not sure if that is needed in the other ones.
- Since there are major changes in the webpage I navigated it. Here my findings:
    - Home page: the picture of MW over Simonyi is heavy, maybe a downgrade in resolution makes it appear easily. The picture on "Enter telescope spaces" can be cropped to have the same size as the "Safety systems" box. The picture for AuxTel is missing.
    - Safety is now so much intuitive, thanks!. The link to the last box "Enter telescopes safely"  points to GIS.
    - Thank you al ot for this change: AuxTel/AuxTel-troubleshooting/ATCS-Troubleshooting/ -->  AuxTel/Troubleshooting/ATCS. In theory is the same, but in reality it is so much intuitive.

File: Daytime-Nighttime-Interactions/announce-component-usage.rst (hunk 3 above)
Comment (isotuela) [reply Daytime-Nighttime-Interactions/announce-component-usage.rst:43]: Changed also the rest of the sections to ========, and in other documents within Daytime-Nighttime-Interactions that were using old templates.

File: Observing-Interface-Setup/dds-interface.rst (hunk 4 above)
Comment (isotuela) [reply Observing-Interface-Setup/dds-interface.rst]: Have replaced it in some other docs in the same folder.

File: Simonyi/Non-Standard-Operations/ComCam/comcam-cooldown.rst (hunk 6 above)
Comment (isotuela) [reply Simonyi/Non-Standard-Operations/ComCam/comcam-cooldown.rst:40]: Several more of this outdated second-level headings have been modified.

**Comment by isotuela** [comment]
Hi Karla!
Thank you so much for your prompt and thorough review!
I have implemented your requests and some related to the headlines. But I can't make the shape of the image under "Enter telescope spaces" to match that of "Safety Systems". I will leave this issue and include it in this week's ticket, that will also fix some labels and build errors. For now, it's better just to merge.

**Pull request merged**